kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -47,7 +47,16 @@ def on_update(doc: Document, method: str | None = None) -> None:
         "modrNm": record.modified_by,
         "modrId": split_user_email(record.modified_by),
     }
-    headers = build_headers(company_name, record.branch)
+    try:
+        headers = build_headers(company_name, record.branch)
+    except Exception as e:
+        frappe.log_error(message=str(e), title="Header Building Error")
+        headers = {
```

Comment:
Building the headers is strictly handled using the build_headers() function. Avoid hard-coding header information as it is critical for the communication with the etims servers.